validator/delphi_validator/validate.py

```diff
@@ -20,7 +20,10 @@ def __init__(self, params):
         Arguments:
             - params: dictionary of user settings; if empty, defaults will be used
```

Comment:
Clarify that the expected params are as read from the params file, not just the `validation` section

Reply:
Done (I think)